test/unicode-spec.js

```diff
@@ -64,6 +77,32 @@ describe("unicode", function() {
 
                 return result;
             },
+
+            toBuild: function(actual, settings) {
+                const usedSettings = settings ? settings : defaultSettings;
+
+                const result = {
+                    pass: true,
+                    message: () => "'" + actual + "' succeeded in building",
+                };
+
+                expect(actual).toParse(usedSettings);
+
+                try {
+                    _getBuilt(actual, settings);
+                } catch (e) {
+                    result.pass = false;
+                    if (e instanceof ParseError) {
+                        result.message = "'" + actual + "' failed to " +
+                            "build with error: " + e.message;
+                    } else {
+                        result.message = "'" + actual + "' failed " +
+                            "building with unknown error: " + e.message;
+                    }
+                }
+
+                return result;
+            },
```

Comment:
Sad to see all this code duplication. I guess more motivation to work on #1126! 😄